tests/utils.js

```diff
@@ -371,13 +371,14 @@ const deprecated = (name, replacement) => {
   }
 };
 
-const waitForSettingsUpdateLogs = (type) => {
+const waitForSettingsUpdateLogs = (type, wdio) => {
   if (type === 'sentinel') {
     return module.exports.waitForLogs(
       'sentinel.e2e.log',
       /Reminder messages allowed between/,
     );
   }
+  if(wdio){return;}
```

Comment:
I'm not sure why we wouldn't want to wait for the api log to update after the settings were reverted. 

Also, I see we are doing this a few places. Let's set a global in the wdio.conf.js in the before hook. 

```
  before: async function () {
    global.isWDIO = true;
    await browser.url('/');
  },
  
 ```
 
 Then when you need to check if we are in WDIO you can do 
 
` if(isWDIO ){ return; }`